modules/kn-service-offline-create.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assemblies:
+//
+// serverless/cli_reference/kn-offline-services.adoc
+
+[id="creating-an-offline-service_{context}"]
+= Creating a service using offline mode
+
+.Prerequisites
+
+* {ServerlessOperatorName} and Knative Serving are installed on your cluster.
+* You have installed `kn` CLI.
+
+.Procedure
+
+. In the offline mode, create a local Knative service descriptor file:
++
+[source,terminal]
+----
+$ kn service create event-display \
+    --image quay.io/openshift-knative/knative-eventing-sources-event-display:latest \
+    --target ./ \
+    --namespace test
+----
++
+[source,terminal]
+----
+Service 'event-display' created in namespace 'test'.
+----
++
+* The `--target ./` flag enables the offline mode and specifies `./` as the directory for storing the new directory tree.
++
+If you do not specify an existing directory, but use a filename, such as `--target my-service.yaml`, then no directory tree is created. Instead, only the service descriptor file `my-service.yaml` is created in the current directory.
++
+The filename can have the `.yaml`, `.yml`, or `.json` extension. Choosing `.json` creates the service descriptor file in the JSON format.
++
+* The `--namespace test` option places the new service in the `test` namespace.
++
+If you do not use `--namespace`, and you are logged in to an OpenShift cluster, the descriptor file is created in the current namespace. Otherwise, the descriptor file is created in the `default` namespace.
+
+. Examine the created directory structure:
++
+[source,terminal]
+----
+$ tree ./
+./
+└── test
+    └── ksvc
+        └── event-display.yaml
+
+2 directories, 1 file
+----
++
+* The current `./` directory specified with `--target` contains the new `test/` directory, named after the specified namespace.
+* The `test/` directory contains the `ksvc` directory, named after the resource type.
+* The `ksvc` directory contains the descriptor file `event-display.yaml`, named according to the specified service name.
```

Comment:
For these three description bullets, WDYT about adding "that is" before "named" so that it is more clear what you are referring to? Also, removing the comma makes it easier, for example, to see that the new `test/` directory is named after the specified namespace (and not the current `./` directory). For example:

> The current `./` directory specified with `--target` contains the new `test/` directory that is named after the specified namespace.